docs/howto/operate/new-cluster/new-cluster.md

```diff
@@ -30,6 +30,16 @@ The _minimum_ inputs this file requires are:
   Primary identifier to 'group' together resources.
 - `project_id`: GCP Project ID to create resources in.
   Should be the id, rather than display name of the project.
+- `regional_cluster`: Set to true to provision a [GKE Regional
+  Highly Available cluster](https://cloud.google.com/kubernetes-engine/docs/concepts/regional-clusters).
+  Costs ~70$ a month, but worth it for the added reliability for most
+  cases except when cost saving is an absolute requirement. Defaults
+  to `true`.
+- `zone`: Zone where cluster nodes and filestore for home directory
+  are created.
+- `region`: Region where cluster master (if `regional_cluster` is
+  `true`) is run, as well as any storage buckets created with
+  `user_buckets`.
```

Comment:
What do you think about adding these extra inputs (`regional_cluster`, `zone`, `region`) to the example below too: 

https://github.com/2i2c-org/infrastructure/blob/1b45e2fe7f1dce300be12996622e3786aeccc054/docs/howto/operate/new-cluster/new-cluster.md?plain=1#L46-L52

Also, I just noticed there are some extra  ` ```` ` there too

Reply:
'the extra `````` matches this line, so it is correct :) https://github.com/2i2c-org/infrastructure/blob/1b45e2fe7f1dce300be12996622e3786aeccc054/docs/howto/operate/new-cluster/new-cluster.md#L26

Reply:
Ah, right! Thanks @sgibson91 <3

Reply:
@GeorgianaElena added these additional variables to the example!